src/Avalonia.Controls/Primitives/AccessText.cs

```diff
@@ -82,17 +82,11 @@ public override void Render(DrawingContext context)
         /// Creates the <see cref="FormattedText"/> used to render the text.
         /// </summary>
         /// <param name="constraint">The constraint of the text.</param>
+        /// <param name="text">The text to generated the <see cref="FormattedText"/> for.</param>
```

Comment:
Non-gramatical comment here too. You can use `/// <inheirtdoc/>` here to prevent copy-pasta errors.